underlay/js/bigipreport.js

```diff
@@ -1061,6 +1086,9 @@ function populateSearchParameters(updatehash) {
                 case 'irules':
                     showiRules(updatehash);
                     break;
+		    case 'policy':
```

Comment:
Replace tab here too. You can replace using regex by replacing `\t` with `    ` (space x4).

Reply:
I see a few more but won't comment on this again to keep the review neat and tidy.